bin/rancher-rc-deps

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/bash
```

Comment:
All of the logic looks fine but nits on formatting. Can you make this look like the rest of the shell scripts in this dir?

- capital var names
- surrounded with "${}"
- 4 space indentation
- update the she bang to "#!/bin/sh" (I don't see any bash specific code)
- single "[", "]" in "if" statements